### DISCUSSION TITLE
Ban Appeal?

### DISCUSSION THREAD

**Discussion by ashymcwashyyy** [ldtteam/minecolonies#10006] [General]
I am so, so sorry if this is not where you ask this. I just saw a comment about this previously and didn't know where else to go. My discord account JUST now got hacked, but I was able to reset my pass, changing it to a safer one, and enabled 2factor auth, and really enjoyed finding other peoples creations on the server. Would there be any chance I could be unbanned?

**Comment by MotionlessTrain** [comment]
> I am so, so sorry if this is not where you ask this. I just saw a comment about this previously and didn't know where else to go. My discord account JUST now got hacked, but I was able to reset my pass, changing it to a safer one, and enabled 2factor auth, and really enjoyed finding other peoples creations on the server. Would there be any chance I could be unbanned?

What is your discord name?

**Reply by ashymcwashyyy** [reply]
ashymcgay, uid is 809179239487504394

**Reply by MotionlessTrain** [reply]
You should be able to access the discord server again!

**Reply by ashymcwashyyy** [reply]
thanks so much! appriciated!!!
